src/mcpbr/profiler.py

```
@@ -144,15 +144,15 @@ def sample_memory(self) -> None:
             sample = MemorySample(
-                timestamp=datetime.now(timezone.utc),
+                timestamp=datetime.now(UTC),
                 rss_mb=memory_info.rss / 1024 / 1024,
                 vms_mb=memory_info.vms / 1024 / 1024,
             )
             self.memory_samples.append(sample)
         except ImportError:
             # psutil not available, disable memory profiling
             self.enable_memory_profiling = False
-        except Exception:
+        except Exception:  # noqa: S110 -- best-effort memory sampling; non-critical telemetry
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)